frontend/src/components/configurate-podcast/configurate-podcast.tsx

```diff
@@ -9,18 +9,46 @@ const ConfiguratePodcast: React.FC = () => {
   const { id } = useParams<PageParams>();
   const dispatch = useDispatch();
 
+  const { podcast } = useAppSelector(({ configuratePodcast }) => ({
+    podcast: configuratePodcast.podcast,
+  }));
+
   const isEdit = Boolean(id);
 
-  const handleSubmit = (payload: PodcastFormPayload): void => {
-    dispatch(configuratePodcastActions.create(payload));
+  const handleFormSubmit = (payload: PodcastFormPayload): void => {
+    isEdit ? dispatch(configuratePodcastActions.edit(payload)) : dispatch(configuratePodcastActions.create(payload));
   };
 
+  const mapPodcastToFormPayload = (podcast: Podcast): PodcastFormPayload => ({
+    name: podcast.name,
+    description: podcast.description,
+    image: null,
+  });
+
+  useEffect(() => {
+    if (isEdit) {
+      dispatch(configuratePodcastActions.loadPodcast(Number(id)));
+    }
+  }, []);
+
+  useEffect(() => {
+    if (isEdit) {
+      dispatch(configuratePodcastActions.loadPodcast(Number(id)));
+    }
+  }, []);
+
   return (
     <div className={styles.podcast}>
       <h2>
         {isEdit ? 'Edit' : 'Create'} Podcast {id ?? ''}
       </h2>
-      <ConfiguratePodcastForm onSubmit={handleSubmit} />;
+
+      {(isEdit)
+        ? podcast && <ConfiguratePodcastForm
+          onSubmit={handleFormSubmit}
+          payload={mapPodcastToFormPayload(podcast)}/>
+        : <ConfiguratePodcastForm onSubmit={handleFormSubmit}/>
+      }
```

Comment:
```suggestion
      <ConfiguratePodcastForm
          onSubmit={handleFormSubmit}
          payload={isEdit ? mappedPodcast : null}
       />
```
mappedPodcast - is the result of calling mapPodcastToFormPayload